process/interface.go

```diff
@@ -184,7 +184,7 @@ type PreProcessorsContainerFactory interface {
 	Create() (PreProcessorsContainer, error)
 }
 
-// PreProcessorsContainer defines an PreProcessors holder data type with basic functionality
+// IntermediateProcessorContainer defines an PreProcessors holder data type with basic functionality
```

Comment:
The name is updated but the comment is still related to `PreProcessors`

Reply:
done